include/openenclave/bits/sgx/sgxproperties.h

```diff
@@ -28,6 +28,13 @@ typedef struct _oe_sgx_enclave_image_info_t
 /* Max number of threads in an enclave supported */
 #define OE_SGX_MAX_TCS 32
 
+#ifdef OE_WITH_EXPERIMENTAL_EEID
+/* This is the maximum size SGX (currently) allows (64GB). When signing EEID
+ * base images we don't know the size that the final image will have, so we
+ * chose this maximum. */
+#define EEID_SECS_SIZE 0x1000000000
```

Comment:
This doesn't need to be a publicly exposed definition, could you move it into the include/openenclave/internal/eeid.h header instead?

Also, consider `EEID_ELRANGE` instead; the `SECS` is a well-known data structure in SGX and the current name implies that it is somehow set to a 64GB size, which makes for a confusing read.

Reply:
Sure, done! Just in case: Yes, this is indeed the size that is put into the SECS (`sgx_secs_t` via `enclave_size`) and it is set to 64GB.